lib/graphql/schema/object.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "graphql/query/null_context"
```

Comment:
Is there a better place for me to load this? Otherwise I can't have null context default params.